src/casp/bq_scripts/anndata_to_avro.py

```diff
@@ -297,10 +382,26 @@ def find_max_index(client, project, dataset, table, column):
     return max_id
 
 
-def process(input_file, cas_cell_index_start, cas_feature_index_start, avro_prefix, project, dataset, load_uns_data):
+def process(
+    input_file,
+    cas_cell_index_start,
+    cas_feature_index_start,
+    prefix,
+    project,
+    dataset,
+    load_uns_data,
+    original_feature_id_lookup=ORIGINAL_FEATURE_ID_LOOKUP_DEFAULT,
+    uns_meta_keys: t.List = None,
```

Comment:
Rename `uns_meta_keys` here and elsewhere to something more descriptive, e.g. `included_adata_uns_keys`.